### DISCUSSION TITLE
Blobfuse2 health-monitor works with direct blobfuse2 command, but with mount -a gives error executible 'bfusemon' not find in $PATH

### DISCUSSION THREAD

**Discussion by ravina199** [Azure/azure-storage-fuse#1089] [Q&A] [answered]
Hi All,

I'm using Blobfuse2 v2.0.2. 
When I run below blobfuse2 command ,  

Blobfuse2 mount_path --config-file=filename.yml

the health-monitor logs are created successfully. But when I run mount -a(using fstab entry), the log files are not created. And on logs i can see error :

LOG_ERR [health-monito.go (113)] failed to start health monitor [exec: "bfusemon" executible file not found in $PATH]

Please assist.

**Comment by souravgupta-msft** [comment, marked as answer]
Hi @ravina199. 
Thanks for reaching out. We were able to reproduce this.
`LOG_ERR [health-monito.go (113)] failed to start health monitor [exec: "bfusemon" executible file not found in $PATH]`
The reason for the above error is that /etc/fstab runs as a system service and does not have access to the $PATH environment variable.
As an alternate solution, you can create a script like below and run it through /etc/fstab.

```
#!/bin/bash
export PATH="$PATH"
blobfuse2 mount_path --config-file=filename.yml
```

Let me know if this works out for you.
